content/news/2023.pml

```diff
@@ -54,6 +54,29 @@ Older News: <%=reflist%>
 </div>
 -->
 
+<!----------------------------------------------------------->
+<div class="news-item" id="summit-2023-cfp">
+    20 February 2023: <a href="../Foundation/Summit/2023/cfp">Call For Presentations</a> for the seL4 Summit 2023
+</div>
+<div>
+  <a href = "../Foundation/Summit">
+    <img src="/images/sel4-summit-logo.svg"
+    style="width: 15%; float:right"
+    alt="seL4 summit" />
+  </a>
+  <h3 class="highlight">
+   Call For Presentations for the seL4 Summit 2023
```

Comment:
You could also add links to CfP and to summit page in that title too. I know it's everywhere now, but it's better than the opposite :)